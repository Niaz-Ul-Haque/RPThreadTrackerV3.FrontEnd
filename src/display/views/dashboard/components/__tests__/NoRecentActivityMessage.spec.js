// #region imports
import React from 'react';
import { shallow } from 'enzyme';
import NoRecentActivityMessage from '../NoRecentActivityMessage';
// #endregion imports

const createTestProps = (propOverrides) => ({
	...propOverrides
});

describe('rendering', () => {
	describe('snapshots', () => {
		it('should render valid snapshot', () => {
			const props = createTestProps();
			const jsx = <NoRecentActivityMessage {...props} />;
			const element = shallow(jsx);
			expect(element).toMatchSnapshot();
		});
	});
});
