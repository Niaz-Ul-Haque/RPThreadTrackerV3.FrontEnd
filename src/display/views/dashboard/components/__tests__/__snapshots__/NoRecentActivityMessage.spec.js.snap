// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering snapshots should render valid snapshot 1`] = `
<div
  className="dashboard-card-center-message"
>
  You do not owe a reply on any of your threads! Nice job!
</div>
`;
